LineCounter sample no longer builds

When I was running `dotnet build eng/services.proj -p:ServiceDirectory=*`, I found that samples/linecounter does not build anymore (our CIs exclude samples) because it uses a public `CloudEvent` type, which was made internal and renamed to `CloudEventInternal` in the referenced EventGrid project.

> C:\src\azure-sdk-for-net\samples\linecounter\Controllers\HomeController.cs(70,56): error CS0246: The type or namespace name 'CloudEvent' could not be found (are you missing a using directive or an assembly reference?) [C:\src\azure-sdk-for-net\samples\linecounter\LineCounter.csproj]

--- a/samples/linecounter/Controllers/HomeController.cs
+++ b/samples/linecounter/Controllers/HomeController.cs
@@ -3,6 +3,7 @@ using System;
 using System.Collections.Generic;
 using System.Text;
 using System.Threading.Tasks;
+using Azure.Messaging;
 using Azure.Messaging.EventGrid;
 using Azure.Messaging.EventHubs;
 using Azure.Messaging.EventHubs.Producer;

